pkgs/development/libraries/openfec/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchzip, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "openfec";
+  version = "1.4.2";
+
+  src = fetchzip {
+    url = "http://openfec.org/files/openfec_v1_4_2.tgz";
+    sha256 = "sha256:0c2lg8afr7lqpzrsi0g44a6h6s7nq4vz7yc9vm2k57ph2y6r86la";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  configurePhase = ''
+    mkdir build
+    cd build
+    cmake .. -DDEBUG:STRING=OFF
+  '';
+
+  installPhase = ''
+    # This is pretty horrible but sadly there is not installation procedure
+    # provided.
+    mkdir -p $dev/include
+    cd ..
+    cp -R src/* $dev/include
+    find $dev/include -type f -a ! -iname '*.h' -delete
```

Comment:
```suggestion
    find .. -type f -a ! -iname '*.h' -print0 | xargs -0 install -D --target $dev/include
```

Reply:
mhm ^ that might not work if you have subdirectories.

Reply:
There indeed are subdirectories, sadly